docs/axelar/intro-to-axelar.md

```diff
@@ -0,0 +1,103 @@
+# Axelar Integration
+
+The Axelar-XRPL integration enables messages to pass between the XRPL and other Axelar-supported blockchains. This integration enables the transfer of XRP and XRPL-issued tokens to other chains and back; this also works for briding non-XRPL tokens from their native chains to XRPL.
+
+In addition to cross-chain features, the Axelar Amplifier network utilizes tickets to enable multiple in-flight transactions to improve throughput.
+
+At a high-level, there are two main components that enable the Axelar-XRPL integration. On the XRPL-side, a multisig account is created; this account handles all incoming and outgoing transactions to the Axelar network by:
+
+- Holding and releasing assets as native XRP assets are moved off and on-chain.
```

Comment:
unclear suggest changing to
 For outbound transactions from XRPL -> Axelar -> another chain

-  locking XRPL-issued assets and asking the Axelar network to issue a wrapped asset on the Axelar network to send to the other chain
- burning non-XRPL wrapped asset on the XRPL and asking the Axelar network to unlock the non-XRP asset on the other chain

 For inbound transactions from Another chain  -> Axelar -> XRPL

- minting a non-XRPL wrapped asset on the XRPL after the original non-XRP asset has been locked on the other chain
- Unlocking the XRP-issued asset after the wrapped version of the asset has been burnt on the other chain and the Axelar network.